docs/traffic-policy/concepts/cel-interpolation.mdx

```diff
@@ -0,0 +1,90 @@
+# CEL Interpolation
+
+**CEL Interpolation** allows you to write dynamic configurations for traffic policy actions. It enables expressions to be embedded within string fields of a configuration object, making the traffic policy system more flexible and powerful.
```

Comment:
```suggestion
**CEL Interpolation** allows you to write dynamic configurations for Traffic Policy Actions. It enables expressions to be embedded within string fields of a configuration object, making the traffic policy system more flexible and powerful.
```

Reply:
are we capitalizing these things in our docs? Actions? Traffic Policy? etc?

Reply:
Yes, we should be.